src/monitoring.c

```diff
@@ -211,6 +213,17 @@ static void handle_client(struct monitoring *monitoring, int fd)
 							status_string[monitoring->disciplining.status]
 						)
 					);
+
+					json_object_object_add(disciplining, "current_phase_convergence_count",
+						json_object_new_int(monitoring->disciplining.current_phase_convergence_count
+						)
+					);
+
+					json_object_object_add(disciplining, "valid_phase_convergence_threshold",
+						json_object_new_int(monitoring->disciplining.valid_phase_convergence_threshold
+						)
+					);
+
```

Comment:
Could you add Progress value regarding a stat that would be returned by minipod ?